internal/server/handlers.go

```diff
@@ -446,6 +419,18 @@ func (a *API) ListGrants(c *gin.Context, r *api.ListGrantsRequest) ([]api.Grant,
 	return results, nil
 }
 
+// TODO: remove after deprecation period
```

Comment:
Let's create a ticket to track this or add it to an existing one so we don't forget.

Reply:
Maybe I should remove them in this PR?  At the very least I should update the CLI to use the new endpoints, and mark the Client methods deprecated as well.